### ISSUE TITLE
[06/11/2020 - 06/11/2020] [Rotkreuz, Switzerland] Zug Crypto Valley

### ISSUE THREAD

**Issue by ashleynear** [nearprotocol/near-events#14]
**What is the event?**
https://www.cryptovalleyconference.com/index

SETTING THE FOUNDATION FOR THE FUTURE OF BLOCKCHAIN

The 3rd edition of the Crypto Valley Conference is coming up with two days of in-depth discussions on the current state and future of blockchain technology.

50+ presentations from global industry leaders on 2 stages
Covering topics on technology, economy & finance and legal & regulation

**Where is the event?**
Rotkreuz, Switzerland

**When is the event?**
06/11/2020 -06/12/2020

**Who are the attendees and how many are expected?**
700+ Attendees from start-ups, corporate, academia and governments

**Describe how you'd like NEAR to ideally participate**
SPEAK! 

**Describe alternatives you'd be open to**
Attend

**Describe what the goal of the event is**

*Bonus: describe a potential goal for NEAR at this event.*
- Speak in front of the press

**Additional Context**
https://www.cryptovalleyconference.com/technology-call-for-papers
https://www.cryptovalleyconference.com/docs/Sponsoring_Dossier_Crypto_Valley_Conference_2020.pdf

**Contact**
contact.general@cryptovalleyconference.com

**Comment by ashleynear** [comment]
<----https://www.foxnews.com/health/switzerland-bans-all-events-over-1000-people-coronavirus-nine-c

TLDR: Switzerland had banned events with more than 1k people until March 15th, 2020.

This event has 700+....which makes it iffy ```In the case of public or private events at which fewer than 1000 people would gather, event organizers must carry out a risk assessment in conjunction with the competent cantonal authorities to decide whether or not the event can be held," ```

Potentially **Blocked** on choices until we know if this will continue after March 15th.

**Comment by ashleynear** [comment]
> For @alexbreines review

##NEAR would like to attend this event for the speaking/press opportunity.



**What is the event?**
https://www.cryptovalleyconference.com/index | Crypto Valley Conference

**Who is the event for?**
700+ Attendees from start-ups, corporate, academia and governments

**When is the event?**
06/11/2020 -06/12/2020

**How is the event structured?**
A conference

**What type of event is it for NEAR?**
Speaking opportunity

**How much would it cost to do?**
Ticket should be free as a speaker, if we get it. 
+ Travel, of about <500 if just Erik & Yessin go, and >1.5K if we send someone from SF


** How much are base tickets?**

~$150 for students
~$300 for startups
~$600 for all others
~$700 for VIP

** Have we attended before?**
Asking in slack

**Who from NEAR Should attend?**
Illia or Erik & Yessin, maybe all

**Comment by ashleynear** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue #14: [06/11/2020 - 06/11/2020] [Rotkreuz, Switzerland] Zug Crypto Valley](https://trello.com/c/D3Jz1gMv/25-issue-14-06-11-2020-06-11-2020-rotkreuz-switzerland-zug-crypto-valley)
